### PR TITLE
tuned fish forces and som bugfixes

### PR DESCRIPTION
**Pull request by fredrikm**

### CODE REVIEW

File: code/agents.py [hunk 1]
```diff
@@ -67,6 +67,8 @@ def calculate_fish_forces(self):
         for (fish, pos) in neighbours:
 
             magnitude = k / np.linalg.norm(pos - self.position)**power
+            if magnitude > 2:
```
Comment (reinthal) [inline code/agents.py:71]: Can you comment on this line please?

Comment (reinthal) [reply code/agents.py:71]: The way I see it - magnitude is controlled by k and power? I'm guessing a cap on force magnitude could serve some purpose.

Comment (rasake) [reply code/agents.py:71]: This looks like a temporary statement for debugging, should it be removed @fredrikm ?

Comment (fredrikm) [reply code/agents.py:71]: no its capped. In case fishes start on top of eachother the force will be so big that it pushes them apart very far very fast

**Pull request merged**